File: app/src/main/java/edgar/yodgorbek/sportnews/database/SportNewsDatabase.java
package edgar.yodgorbek.sportnews.database;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;
import android.arch.persistence.room.TypeConverters;

import edgar.yodgorbek.sportnews.model.Article;
import edgar.yodgorbek.sportnews.model.Source;

@TypeConverters(SourceTypeConverter.class)
@Database(entities = {Article.class, Source.class}, version = 1, exportSchema = false)
public  abstract class SportNewsDatabase extends RoomDatabase {
    public  abstract SportNewsDao sportNewsDao();
}

File: app/src/main/java/edgar/yodgorbek/yangiliklar/database/SportNewsDatabase.kt
package edgar.yodgorbek.yangiliklar.database

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase
import android.arch.persistence.room.TypeConverters

import edgar.yodgorbek.yangiliklar.model.Article
import edgar.yodgorbek.yangiliklar.model.Source

@TypeConverters(SourceTypeConverter::class)
@Database(entities = [Article::class, Source::class], version = 1, exportSchema = false)
abstract class SportNewsDatabase : RoomDatabase() {
    abstract fun sportNewsDao(): SportNewsDao
}